src/Session.php

```diff
@@ -363,6 +362,7 @@ public function wait($time, $condition = 'false')
      */
     public function resizeWindow($width, $height, $name = null)
     {
+        $this->start();
```

Comment:
technically, these actions are not defined as supported before visiting (see the phpdoc of ``start``)